src/transaction.ts

```diff
@@ -90,7 +90,7 @@ export interface BuildTxArgs {
   errorHandler?: CoinSelectorErrorFn;
 }
 
-export const DEFAULT_SATS_PER_BYTE = 0.1;
+export const DEFAULT_SATS_PER_BYTE = 0.2;
```

Comment:
Why this?

Reply:
Otherwise CI fails with "min relay fee not met, 124 < 130" https://github.com/vulpemventures/ldk/actions/runs/3461702660/jobs/5779643032

Reply:
But I changed this a bit too fast, I thought it would affect just the tests. Maybe this is better https://github.com/vulpemventures/ldk/pull/143/files#diff-daff4cb5ab57a2886a0cfe4dae670b9f75c84e865e6dd147a5ad8976ea51924cR194 ?

Reply:
this tells me our tx estimation may not be right. Which type of script it's failing?

Reply:
ok best to tweak from code, rather than change the constant. Will investigate eventually why estimation is changing